core/templates/base-components/base-content.component.css

```diff
@@ -77,6 +55,31 @@
   width: 168px;
 }
 
+@media screen and (max-width: 600px) {
```

Comment:
What if the screen is wider than 600px, then these classes will be absent, but it doesn't mean that the containers with these classes will not be presented, right?

Another thing, you are changing something super fundamental (based on the css name). I'd recommend not touching it and fixing it in component that you want to fix. Chances are this PR will introduce problems. OTOH, if you are sure you want to do this across the whole website, maybe it's a right place, but then you need to check ~the whole website to verify the changes don't break anything.

Also, @seanlip, I'd say it's the 6th or so PR in the last 2 months I get with these @ media and max-width. We have 77(!) entries of media with max width, some are just crazy (max-width: 1099px O_o, what is 1099px). It looks not consistent and I'm pretty sure causes many inconsistencies. I don't know what to do with this, just bringing this up to you.

Reply:
@nikitaevg I think that's a great callout, thanks for the heads-up. Would you mind filing an issue for this so that I can bring it up with the design team and ask if we can have a more consistent standard here?

Thanks!

Reply:
👍 https://github.com/oppia/oppia/issues/20493

Reply:
Should I make some changes and push a temporary fix for this issue or should we focus on this other issue first?

Reply:
@thiago-monteiro I think, for now, it's fine to tidy this case up, but you'll need to tidy it up correctly. That means providing proof that this works correctly on all sizes and also ensuring that it doesn't break other pages (you'll want to explain how you did that, maybe look at where the CSS class is used).